frontend/src/components/Footer.js

```diff
@@ -145,6 +145,19 @@ const Footer = () => (
             </Stack>
           );
         })}
+        <Stack align={"flex-start"}>
```

Comment:
Can we keep this content defined with rest sitemap, just add/use a flag that rednders component only in the footer and skips sidebar/navbar?

Reply:
Remove comments at L125